packages/astro/src/@types/astro.ts

```diff
@@ -1095,6 +1096,40 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		middleware?: boolean;
+		
+		/**
+		 * @docs
+		 * @name experimental.hybridOutput
+		 * Enable experimental support for opt-out prerendering
+		 *
+		 * To enable this feature, first set `experimental.hybridOutput` to `true` in your Astro config, and set the `output` to `hybrid`.
+		 *
+		 * ```js
+		 * {
+		 *  output: 'hybrid',
+		 * 	experimental: {
+		 *		hybridOutput: true,
+		 * 	},
+		 * }
+		 * ```
+		 * Then add `export const prerender =  false` to any page or endpoint you want to opt-out of prerendering.
```

Comment:
```suggestion
		 * Then add `export const prerender =  false` to any page or endpoint you want to opt-out of pre-rendering.
```
I noticed that we are not totally consistent in our own docs (I'll make a task to go fix that!) but everywhere else online seems to use the hypen for `pre-rendering`. So if that seems right to you, let's make that our standard.